src/Gating.Example/Program.cs

```diff
@@ -128,6 +128,9 @@ internal class SampleGatedRequest : IGatedRequest
 		};
 
 
+		public HashSet<string> CloudContexts => throw new NotImplementedException();
```

Comment:
>public [](start = 2, length = 6)

I think it would be useful to add an example for cloud context support in this program